### DISCUSSION TITLE
Cards not accepted, they are getting declined

### DISCUSSION THREAD

**Discussion by aviral17** [planetscale/discussion#530] [FAQs]
I can't use my Debit Card, Discover or Visa or MasterCard all are declined, why man ??? I want to use Free Tier. Also last month I created project with Prisma PlanetScale, that time they haven't asked for any card, and I successfully used free tier, but now I can't even see my Database/Project which I created last month, All vanished, I mean seriously ??? Why you doing this ??? Please help me

**Comment by jgreet** [comment]
We answered your support request, but to follow up here the card requirement for database creation was added in August. 

We don't receive a lot of details about card declines, you'll have to contact your bank to discover the exact reason.
